modules/cni/cilium/README.md

```diff
@@ -0,0 +1,43 @@
+# Cilium Module
+
+Cilium is a cloud-native solution that provides network connectivity.
+The Cilium module configures the *Container Network Interface* (CNI) necessary to implement the *Kuberentes network model* of your cluster.
+
+[Here is the link to Cilium official repository.](https://github.com/containous/traefik)
+
+## What the base module contains
+
+- **Resources (Traefik ingress):**
+  - **Cilium-agent:** RBAC and definitions of the Cilium-agent deamonset.
+  - **Cilium-operator:** RBAC and definitions of the Cilium-operator deployments.
+  - **Configs:** configmap with the necessary configurations of Cilium.
+  - **Hubble** RBACs and workloads for the Hubble observability platform.
+
+## Flavors
+
+### EKS
+
+- Adds `egress-masquerad-interfaces: eth0` in the Cilium config.
```

Comment:
Here is better to add a notice that by default the configurations expect that in the EKS cluster it will not be installed the AWS CNI Daemonset

Reply:
added the note, there is a standard way to include warnings?

Reply:
This is the time to have one 😄 

We can make an all caps `warning` subheader with maybe the ⚠️ emoji to draw attention to it?